### DISCUSSION TITLE
Search/Filter by multiple tags

### DISCUSSION THREAD

**Discussion by cdmcdermott** [adityatelange/hugo-PaperMod#589] [Ideas]
It would be great if you could search/filter posts by multiple tags e.g.

Filter for all articles related to `Linux` but then filter these down also by `Exercises` to show just linux exercises. 
I know I could achieve this by having three tags `Linux` `Excerises` and `Linux Exercises` but would be cleaner if someone could first see all exercises and then filter the exercises for just `Linux`

**Comment by pixelfabs** [comment]
Static solution in https://gronskiy.com/posts/filtering-posts-over-multiple-taxonomies-hugo/

TLDR: requires an external solution to generate pages with multiple tags. In the article above a Go module is used. Another solution is a Javasript libary (outdated and doesn't have permalinks). Pro's: narrow down search. Cons: 1000's of extra pages and 2 filters is the max. 

This is a more relational database type thing, which Hugo is not.
